### PR TITLE
apps/projects: increase max length of tile image name to fix bplan is…

### PR DESCRIPTION
**Pull request by fuzzylogic2000**
…sue in mB (https://github.com/liqd/a4-meinberlin/issues/4356)

It will appear in the bplans error messages automatically because it gets the length from a4
https://github.com/liqd/a4-meinberlin/blob/e8b129ec7daac2890af6e76b95269a78c5a707d8/meinberlin/apps/bplan/serializers.py#L42

### CODE REVIEW

File: adhocracy4/projects/models.py [hunk 1]
```diff
@@ -241,9 +242,11 @@ class Project(ProjectContactDetailMixin,
         ),
         upload_to='projects/tiles',
         blank=True,
-        max_length=300)
+        max_length=512
+    )
```
Comment (phillimorland) [inline adhocracy4/projects/models.py:246]: will check this again mB tiles in morning as if it's same test then might break styling

Comment (fuzzylogic2000) [reply adhocracy4/projects/models.py:246]: It's the name of the image, which is hopefully not displayed anywhere...

**Review by phillimorland — APPROVED** [review]
looks good, doesn't work with mB because of node but looking fine on a+

**Comment by fuzzylogic2000** [comment]
> looks good, doesn't work with mB because of node but looking fine on a+

Yeah, we need to cherry-pick this into an extra branch to actually get it into mB without updating all a4.

**Pull request merged**